src/components/AboutUsSection/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { Link } from "../../styles";
+import { Container, DownButonContainer, DownButton, GoDownIcon, TextArea } from "./style";
+
+function AboutUs() {
+    return (
+      <Container>
+        <TextArea>
+          <h1>Hi!</h1>
+          <p>
+            We are Beata & Natalia, and this is our portfolio web!
+            <br />
+            We've been friends since hight school. We've always loved creating
+            projects together.
+            <br /> A year ago we discover a new passion - designing mobile apps.
+            <br />
+            That's how we started our jouney with UX/UI design and coding. We're
+            glad that you visit us!
+          </p>
+        </TextArea>
+        <DownButonContainer>
+          <Link smooth to="/#projecttilesection">
+            <DownButton>
+              <GoDownIcon src="assets/go_down_icon.png" alt="down arrow" />
```

Comment:
It works but i'm wondering if simply importing it as a module help with creating bundle?

Reply:
Can you explain in what way it can help?

Reply:
<img width="405" alt="image" src="https://user-images.githubusercontent.com/26671751/185844394-c76a1192-6688-4a2d-835c-297fb2f27811.png">
vs
<img width="401" alt="image" src="https://user-images.githubusercontent.com/26671751/185844483-cc083552-0761-4695-b6e7-7b202f811ccd.png">

Reply:
I meant CRA to avoid additional network requests add to the src data URI instead of the path (less than 10,000 bytes).

this is of course a cool feature of CRA and your way of displaying images is good. Additionally, importing images as modules helps catch potentially deleted files/typos. It's just a suggestion of how you can achieve the same in other way in this specific boilerplate :)

<img width="740" alt="image" src="https://user-images.githubusercontent.com/26671751/185846452-233a7f0d-d60e-49ed-9eab-3d8f3b92bdfb.png">

Reply:
I remember one more additional feature. Result filenames include content hashes so you don’t need to worry about browsers caching their old versions.